### ISSUE TITLE
Fix cucumber upgrade issues

### ISSUE THREAD

**Issue by h-bragg** [graze/cucumber-rest-bdd#22]
Some changes have happening in cucumber that deprecates some language features:

Currently getting the errors:

```
Named capture groups are not supported. See https://github.com/cucumber/cucumber/issues/329 (Cucumber::CucumberExpressions::CucumberExpressionError)
```

You can get a working docker image by using the `0.5.3` tag, the gem package should work as expected.

**Comment by h-bragg** [comment]
Fixed in #26

**Comment by biggianteye** [comment]
🎉
